docs/android/emulator/device-manager.md

```diff
@@ -0,0 +1,434 @@
+ms.custom: video
+ms.date: 11/18/2021
+no-loc: ["user.config"]
+---
+
+# Managing virtual devices with the Android Device Manager
+
+This article explains how to use the Android Device Manager to create and configure Android Virtual Devices (AVDs) that emulate physical Android devices. You can use these virtual devices to run and test your app without having to rely on a physical device.
+
+> [!IMPORTANT]
+> Enable hardware acceleration for the android devices. For more information, see [Hardware Acceleration for Emulator Performance](hardware-acceleration.md).
+
+<!--
+::: zone pivot="windows"
+-->
+
+## Android Device Manager on Windows
+
+You use the Android Device Manager to create and configure an Android Virtual Devices (AVD) that run in the [Android Emulator](../deployment/debug-on-emulator.md). Each AVD is an emulator configuration that simulates a physical Android device. This makes it possible to run and test your app in a variety of configurations that simulate different physical Android devices.
+
+:::image type="content" source="media/device-manager/win/01-devices-dialog-sml.png" alt-text="Screenshot of the Android Device Manager in the Devices tab." lightbox="media/device-manager/win/01-devices-dialog.png":::
+
+## Requirements
+
+To use the Android Device Manager, you'll need the following items:
+
+- Visual Studio 2022 version 17.1 Preview 1. Visual Studio Community, Professional, and Enterprise editions are supported.
+
+- The Android SDK. Be sure to install the Android SDK at its default location if it isn't already installed: _C:\\Program Files (x86)\\Android\\android-sdk_.
+
+- The following packages must be installed in the:
+
+  - **Android SDK Tools version 26.1.1** or later
+  - **Android SDK Platform-Tools 27.0.1** or later
+  - **Android SDK Build-Tools 27.0.3** or later
+  - **Android Emulator 27.2.7** or later
+
+  These packages should be displayed with **Installed** status as seen in the following screenshot:
+
+  :::image type="content" source="media/device-manager/win/02-sdk-tools.png" alt-text="Installing Android SDK Tools.":::
+
+When you install the **Mobile development with .NET** workload in Visual Studio, and select the optional **.NET MAUI (Preview)** workload, everything is installed for you. For more information on setting up .NET MAUI with Visual Studio, see [Installation](../../get-started/installation.md).
+
+## Open the device manager
+
+Open the Android Device Manager in Visual Studio from the **Tools** menu by pressing **Tools > Android > Android Device Manager**:
+
+:::image type="content" source="media/device-manager/win/03-tools-menu.png" alt-text="Launching the Device manager from the Tools menu.":::
+
+<!-- TODO: Is this still applicable? -->
```

Comment:
No. The min SDK version is 30: Beginning in August 2021, the Google Play Console requires that new apps target API level 30 (Android 11.0) or higher. Existing apps are required to target API level 30 or higher beginning in November 2021.

So no one should need to install API26.